lib/mongoid/scroll/cursor.rb

```diff
@@ -1,17 +1,17 @@
 module Mongoid
   module Scroll
     class Cursor
-      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction
+      attr_accessor :value, :tiebreak_id, :field_type, :field_name, :direction, :include_current
 
       def initialize(value = nil, options = {})
         @field_type, @field_name = Mongoid::Scroll::Cursor.extract_field_options(options)
         @direction = options[:direction] || 1
+        @include_current = options[:include_current] || false
```

Comment:
This will have to be added to the serialization(s) method(s) if #28 ends up being merged.